build.sbt

```diff
@@ -157,39 +162,30 @@ lazy val persistence: Project = (project in file("persistence"))
   )
   .dependsOn(core % "compile->compile;test->test", commonTest % "test")
 
-lazy val persistenceFile: Project = (project in file("persistence-file"))
+lazy val persistenceFile: Project = (project in file("persistence/persistence-file"))
   .settings(buildSettings)
   .settings(
     Seq(
-      name := "persistence-file",
-      libraryDependencies ++= Seq(
-        akka2Actor,
-        akka2Slf4j,
-        pureConfig,
-        akka2Testkit,
-        scalaAsync
-      ) ++ common
+      name := "elasticmq-persistence-file",
+      libraryDependencies ++= Seq(pureConfig) ++ common
     )
   )
-  .dependsOn(persistence, commonTest % "test")
+  .dependsOn(persistenceCore, commonTest % "test")
 
-lazy val persistenceSql: Project = (project in file("persistence-sql"))
+lazy val persistenceSql: Project = (project in file("persistence/persistence-sql"))
   .settings(buildSettings)
   .settings(
     Seq(
-      name := "persistence-sql",
+      name := "elasticmq-persistence-sql",
+      publishArtifact := true,
```

Comment:
I think this is the default? Though probably we should set it to `false` for the aggregating project, `persistence`?

Reply:
fixed